m3-libs/libm3/src/arith/POSIX/Math.i3

```diff
@@ -130,21 +130,21 @@ CONST
 <*EXTERNAL*> PROCEDURE hypot (x, y: LONGREAL): LONGREAL;
 (* returns sqrt (x*x + y*y). *)
 
-<*EXTERNAL*> PROCEDURE cabs (z: Complex): LONGREAL;
+PROCEDURE cabs (z: Complex): LONGREAL;
```

Comment:
I think this doesn't always work. For some reason.
That you need to use <* external Math__cabs *> and such.

Reply:
Oh I see what you did..normally I don't like that, but since these are structs, yeah, you need another layer, like you probably have (I didn't finish reading). Or pass by ref?

Reply:
If you pass z as READONLY does that imply pointer?
I don't remember.